src/basic/random-util.c

```diff
@@ -33,6 +33,66 @@
 
 int rdrand(unsigned long *ret) {
 
+        /* So, you are a "security researcher", and you wonder why we bother with using raw RDRAND here,
+         * instead of sticking to /dev/urandom or getrandom()?
```

Comment:
`you are a "security researcher", and ` 
I would just drop that. It's unnecessary confrontional and doesn't really help getting the message across.